src/ui/card/Card.tsx

```diff
@@ -0,0 +1,12 @@
+import { CardContent, Paper } from '@mui/material';
+
+import { styles } from './Card.styles';
+import { CardProps } from './Card.types';
+
+export const Card = ({ children }: CardProps) => {
+  return (
+    <Paper elevation={styles.elevation}>
```

Comment:
To elevation nie właściwie nie jest stylem, tylko parametrem `Paper`. Więc można to śmiało wpisać bezpośrednio. W stylach trzymamy to co jest bezpośrednio związane z CSSem i stylowaniem